### ISSUE TITLE
Conversion reports July, 2020-07-13, 2020-07-14, 2020-07-20, 2020-07-22, 2020-08-01

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/data#157]
Tests 2020-07-14 at http://dev4.species.id:2204/col_plus/info/ac

**Comment by yroskov** [comment]
General stats:

                                    2020-06-04 vs 2020-07-13 
taxonomic databases 171 164 = **lost 7** 
living spp 1,829,672  1,831,489  
extinct species 38,145 38,121 = **lost 24**
living infa 159,503 160,035  
extinct infra 589 590  
synonyms 1,710,621 1,691,500 = lost **19,121** 
common names 423,691 3,011 = lost **420,680**

genera 175,459 184,526 
families 10,758  12,267 
orders 1,398 1,643 
classes 322 387 
phyla 102 104  
kingdoms 7 7

**Comment by yroskov** [comment]
- [ ] **Incorrect GSD titles** (they are back again!)

ColeorrhynchaSF: Coleorrhyncha Species File - should be SF Coleorrhyncha 
ITIS Global: The Integrated Taxonomic Information System -- ITIS Selected
Lace Bugs Database: Lace Bugs Database (Hemiptera: Tingidae)  -- in ac19: Lace Bugs Database (Hemiptera: Tingidae) - _how to do it?_
LepIndex: LepIndex: The Global Lepidoptera Names Index -- LepIndex: The Global Lepidoptera Names Index 
nomen.eumycetozoa.com: Nomenclatural Database of Eumycetozoa (Myxomycota)  -- Nomen.eumycetozoa.com: Nomenclatural Database of Eumycetozoa (Myxomycota) 
OlogamasidBase: Mites GSDs: OlogamasidBase  -- Mites GSD Ologamasidae 
PhytoseiidBase: Mites GSDs: PhytoseiidBase -- Mites GSD Phytoseiidae 
Psyllist: Psylloidea database -- Psyl'list: Psylloidea database 
RhodacaridBase: Mites GSDs: RhodacaridBase  -- Mites GSDs: RhodacaridBase 
TenuipalpidBase: Mites GSDs: TenuipalpidBase  -- Mites GSD Tenuipalpidae 
Tessaratomidae Database -- Tessaratomidae Database: Illustrated catalog of Tessaratomidae
The White-Files  -- The White-Files: Taxonomic checklist of the world’s whiteflies (Insecta: Hemiptera: Aleyrodidae) 
World Plants: World Plants: Synonymic Checklists of the Vascular Plants of the World -- World Plants: Synonymic Checklists of the Vascular Plants of the World

**Comment by yroskov** [comment]
- [ ] **World Spider Catalog** id29
Should be:
Number of living species 	47,391
Number of living infraspecific taxa 	571
Number of extinct species: 	0
Number of extinct infraspecific taxa: 	0
Number of synonyms: 	7,110
Number of common names: 	0
Total number of names: 	55,072 

Now (wrong):
Number of living species 	47,714
Number of living infraspecific taxa 	568
Number of extinct species: 	0
Number of extinct infraspecific taxa: 	0
Number of synonyms: 	0
Number of common names: 	0
Total number of names: 	48,282

**Comment by yroskov** [comment]
- [x] Release dates have been checked against earlier corrections: Incorrect Release Date in 14 GSDs #120 https://github.com/CatalogueOfLife/data/issues/120. All checked dates are OK.

**Comment by yroskov** [comment]
- [ ] **Incorrect data in Archaea & Bacteria** 
Now:
Archaea • **754** of 502 est. living spp (100%) • **ITIS, ITIS Global**
Phylum Crenarchaeota • **116** living spp • ITIS, ITIS Global
**Class Thaumarchaeota** • 0 living spp • ITIS
Class Thermoprotei • 116 living spp • ITIS, ITIS Global
Phylum Euryarchaeota • **638** living spp • ITIS, ITIS Global


Was in 2020-06-04:
Archaea • **377** of 502 est. living spp (75%) • ITIS Selected
Phylum Crenarchaeota • **58** living spp • ITIS Selected
Class Thermoprotei • 58 living spp • ITIS Selected
Phylum Euryarchaeota • **319** living spp • ITIS Selected

**Comment by gdower** [comment]
>     * [x]  **Incorrect GSD titles** (they are back again!)
> 
> 
> ColeorrhynchaSF: Coleorrhyncha Species File -- should be SF Coleorrhyncha: Coleorrhyncha Species File
> ITIS Global: The Integrated Taxonomic Information System -- ITIS Selected: The Integrated Taxonomic Information System
> Lace Bugs Database: Lace Bugs Database (Hemiptera: Tingidae) -- in ac19 a single: Lace Bugs Database (Hemiptera: Tingidae) - _how to do it?_
> LepIndex: LepIndex: The Global Lepidoptera Names Index -- LepIndex: The Global Lepidoptera Names Index
> nomen.eumycetozoa.com: Nomenclatural Database of Eumycetozoa (Myxomycota) -- Nomen.eumycetozoa.com: Nomenclatural Database of Eumycetozoa (Myxomycota)
> OlogamasidBase: Mites GSDs: OlogamasidBase -- Mites GSD Ologamasidae
> PhytoseiidBase: Mites GSDs: PhytoseiidBase -- Mites GSD Phytoseiidae
> Psyllist: Psylloidea database -- Psyl'list: Psylloidea database
> RhodacaridBase: Mites GSDs: RhodacaridBase -- Mites GSDs: RhodacaridBase
> TenuipalpidBase: Mites GSDs: TenuipalpidBase -- Mites GSD Tenuipalpidae
> Tessaratomidae Database -- Tessaratomidae Database: Illustrated catalog of Tessaratomidae
> The White-Files -- The White-Files: Taxonomic checklist of the world’s whiteflies (Insecta: Hemiptera: Aleyrodidae)
> World Plants: World Plants: Synonymic Checklists of the Vascular Plants of the World -- World Plants: Synonymic Checklists of the Vascular Plants of the World
> 
> _These unwanted changes in GSD names make assessment of stats per GSD very difficult (need a lot of manual corrections in alphabetically ordered lists)._ Checks postponed for after GSD names correction.

We fixed this with the metadata patches in the Clearinghouse, but the ac-export doesn't actually contain the fixes, so either the patches aren't getting applied to the metadata or there might be an exporter bug. I adjusted my monthly release SQL to patch the titles.

**Comment by yroskov** [comment]
**Scarabs, 2020-07-01**
Looks pretty good!

**Comment by yroskov** [comment]
**HymIS Pompilidae 2019-09-04**
Data look good

- [ ] YR should ask Alexander about Organization and a list of authors for other HymIS. (Request snt 2020-07-14)

**Comment by yroskov** [comment]
**World Ferns**

**Comment by gdower** [comment]
> **ITIS Archaea & Bacteria**
> 
>     * [x]  **Incorrect data in Archaea & Bacteria**
>       Now:
>       Archaea • **754** of 502 est. living spp (100%) • **ITIS, ITIS Global**
>       Phylum Crenarchaeota • **116** living spp • ITIS, ITIS Global
>       **Class Thaumarchaeota** • 0 living spp • ITIS
>       Class Thermoprotei • 116 living spp • ITIS, ITIS Global
>       Phylum Euryarchaeota • **638** living spp • ITIS, ITIS Global
> 
> 
> Was in 2020-06-04:
> Archaea • **377** of 502 est. living spp (75%) • ITIS Selected
> Phylum Crenarchaeota • **58** living spp • ITIS Selected
> Class Thermoprotei • 58 living spp • ITIS Selected
> Phylum Euryarchaeota • **319** living spp • ITIS Selected
> 
> Now:
> Bacteria • **19,960** of 10,358 est. living spp (100%) • **ITIS, ITIS Global**
> 
> Was:
> Bacteria • **9,980** of 10,358 est. living spp (96%) • ITIS Selected
> 
> _**Stats are doubled, but there are no duplicated species in the tree. What's wrong?**_
> 
>     * [ ]  **Missing references with spp names**
> 
>     * [ ]  **Incorrect credits in the classification** (all to CoL):

This was caused by the old sector still being present with the new sector. Yuri thinks maybe he used replace in assembly, so there might be a bug with replace in assembly.

**Comment by yroskov** [comment]
- [ ] **Unexpected disparities in GSD stats**

Source database,2020-06-04 Beta,,2020-07-13,
3i Auchenorrhyncha: World Auchenorrhyncha Database,**10451**,522,**3210**,498
Collembola.org,**8555**,0,**8671**,0
FLOW: Fulgoromorpha Lists On the WEB,**13356**,356,**13339**,356
Species Fungorum Plus,**147898**,0,**147934**,0
The White-Files: Taxonomic checklist of the world’s whiteflies (Insecta: Hemiptera: Aleyrodidae),1562,0,1565,0
WoRMS Ostracoda: World Ostracoda Database,6307,0,6308,0
WSC: World Spider Catalog,**47391**,0,**47714**,0
WTaxa: Electronic Catalogue of Weevil names (Curculionoidea) ,**84512**,0,**84476**,0

Most problematic (why it's happened?!):
3i Auchenorrhyncha
Collembola.org
Species Fungorum Plus
WSC

**Comment by yroskov** [comment]
**Animal Biodiversity** -- OK
**CoL Management Classification** -- might be, OK

**Comment by yroskov** [comment]
**IRMNG**
_Different Taxonomic Coverage reports. However, it does not mean that taxa are missing. More investigations needed._

2020-07-14:
Chromista - Picozoa, Endohelea, Cryptista, Haptophyta, Acavomonidia
Chromista - Miozoa - Syndinea, Perkinsea, Oxyrrhea, Eucoccidiida, Protococcidiida, Neogregarinida, Acephalidae, Eugregarinida, Againococcidiida, Archigregarinida, Noctilucea, Piroplasmida, Ellobiopsea, Colponemea, Coccidiomorphea, Apicomonadea

Protozoa - Sulcozoa, Nephridiophagidae, Stephanomonadaceae, Paramastigaceae, Jolyaceae, Brasilobiaceae, Metamonada, Loukozoa, Euglenozoa, Calcitarcha
Protozoa - Amoebozoa - Lobosa - Arcellinida - Trichosidae, Plagiopyxidae, Microcoryciidae, Cryptodifflugiidae, Distomatopyxidae, Heleoperidae, Hyalospheniidae, Lamtopyxidae, Lesquereusiidae, Microchlamyiidae
Protozoa - Choanozoa - Rozellidea, Cristidiscoidea, Filasterea, Choanoflagellatea
Protozoa - Amoebozoa - Variosea, Tubulinea, Schizopyrenida, Discosea, Pellitidae

Plantae - Charophyta, Chlorophyta, Glaucophyta, Rhodophyta

2020-06-04:
Chromista - Picozoa, Endohelea, Cryptista, Haptophyta, Acavomonidia
Chromista - Miozoa - Dinophyceae - Gymnodiniales - Polykrikaceae, Pronoctilucaceae, Actiniscaceae, Hemidiniaceae, Tovelliaceae, Warnowiaceae, Kareniaceae
Chromista - Miozoa - Syndinea, Perkinsea, Oxyrrhea, Eucoccidiida, Protococcidiida, Neogregarinida, Acephalidae, Eugregarinida, Againococcidiida, Archigregarinida, Noctilucea, Piroplasmida, Ellobiopsea, Colponemea, Coccidiomorphea, Apicomonadea
Chromista - Miozoa - Dinophyceae - Dinamoebales, Gonyaulacales, Ptychodiscales, Peridiniales, Amphilothales, Brachidiniales, Zooxanthellales, Desmocapsales, Chytriodiniales, Suessiales, Phytodiniales, Adnatosphaeridiaceae, Dollidiniaceae, Chytridiodinidae, Spiromonadaceae, Comparodiniaceae, Gomesiamonadaceae, Stephanelytraceae, Prorocentrales, Dinophysiales, Blastodiniales, Nannoceratopsiales
Chromista - Ochrophyta - Chrysophyceae, Chrysomerophyceae, Bolidophyceae, Xanthophyceae, Raphidophyceae, Phaeothamniophyceae, Eustigmatophyceae, Pinguiophyceae, Picophagophyceae, Phaeophyceae, Dictyochophyceae, Schizocladiophyceae

Protozoa - Sulcozoa, Nephridiophagidae, Stephanomonadaceae, Paramastigaceae, Jolyaceae, Brasilobiaceae, Metamonada, Loukozoa, Euglenozoa, Calcitarcha
Protozoa - Amoebozoa - Lobosa - Arcellinida - Trigonopyxidae, Plagiopyxidae, Trichosidae, Microcoryciidae, Cryptodifflugiidae, Distomatopyxidae, Heleoperidae, Hyalospheniidae, Lamtopyxidae, Lesquereusiidae, Microchlamyiidae
Protozoa - Microsporidia - Minisporea
Protozoa - Choanozoa - Ichthyosporea - Amoebidiales, Dermocystida, Eccrinales
Protozoa - Choanozoa - Rozellidea, Cristidiscoidea, Filasterea, Choanoflagellatea
Protozoa - Amoebozoa - Pelobiontida, Variosea, Tubulinea, Schizopyrenida, Discosea, Pellitidae

Plantae - Charophyta, Chlorophyta, Glaucophyta, Rhodophyta
Plantae - Tracheophyta - Pinopsida - Vojnovskyales FOSSIL-should be deleted

**Comment by gdower** [comment]
There are page break special characters `^L` in the ITIS metadata, which perhaps is interpreted as a tab delimiter by the MySQL importer:

![image](https://user-images.githubusercontent.com/8573609/87583157-2752e600-c6a1-11ea-9dfd-ada12e5dbe27.png)

I removed the hidden special characters in the Clearinghouse metadata, so it should be fixed in the next conversion:

![image](https://user-images.githubusercontent.com/8573609/87584679-7b5eca00-c6a3-11ea-9e6a-5f7fe924e7bf.png)

**Comment by yroskov** [comment]
**Mess in the hierarchy** - mix of classes and orders in Miozoa

![image](https://user-images.githubusercontent.com/28574252/87585406-aa297000-c6a4-11ea-8512-cf032edf8e8c.png)

**Comment by yroskov** [comment]
**Collembola.org**
Number of species and synonyms "illegally" increased - WHY?

Was in 2020-06-04 vs in 2020-07-14
spp 8,555 vs 8,671 = +116
syn 4,709 vs 4,736 = +27

![image](https://user-images.githubusercontent.com/28574252/87589208-82d5a180-c6aa-11ea-9ddc-71d833c5ca77.png)

is:
![image](https://user-images.githubusercontent.com/28574252/87589811-6e45d900-c6ab-11ea-9fc7-ebbd5d7a8e2a.png)

was: 
![image](https://user-images.githubusercontent.com/28574252/87589836-7bfb5e80-c6ab-11ea-9f01-620945dbd3b9.png)

**Comment by gdower** [comment]
Here's the [statistics diff report between the latest monthly and AC2019](https://docs.google.com/spreadsheets/d/1BC0O5-zCJ7Ndl1SNUcZS2lPy1yT-jVAi-XzozzV-nzg/edit#gid=2097276737), although keep in mind that updated GSDs will of course have different statistics. The biggest issue is with missing common names and we're looking into it.

**Comment by gdower** [comment]
Here's a [diff of 25 random species pages from each GSD](http://ower.org/col_diff/2020-07-14/).

**Comment by yroskov** [comment]
After assessment of diff report:

FLOW
FLOW
FLOW
FLOW
FLOW
ICTV_MSL
ICTV_MSL
Parhost
Scarabs
Scarabs
Scarabs
Scarabs
Scarabs
Scarabs
Scarabs
Scarabs
Scarabs
WSC
WSC
LDL   Neuropterida
ELPT
MBB
HymIS
Tessaratomidae   Database
SF Dermaptera
The   Scorpion Files
Gymnodinium
WoRMS   Ctenophora
WoRMS PycnoBase
SF   Isoptera
StaphBase

**Comment by yroskov** [comment]
Diff report 2020-07-14 vs 2020-06-04 at http://ower.org/col_diff/2020-07-14/  

GSDs with missing synonyms:

ICTV_MSL
ICTV_MSL
ICTV_MSL
Parhost
WSC
WSC
Trichomycetes
LDL  Neuropterida
ELPT
MBB
HymIS (it might be OK, because we dropped off Pompelidae from this GSD)
Tessaratomidae Database
SF Dermaptera
The   Scorpion Files
WoRMS Ctenophora
WoRMS   PycnoBase
SF Isoptera

**Comment by gdower** [comment]
Some of these like ICTV_MSL are actually probably fine, but it's the known [ConversionTool natural keys bug](https://github.com/Sp2000/ConversionTool/issues/5):

http://ower.org/col_diff/diff.php?left=5245bebc9585247625d6e46d85ab16e9&right=5245bebc9585247625d6e46d85ab16e9

**Comment by yroskov** [comment]
**CoL 2020-07-20** at http://dev4.species.id:9191/col_plus/info/ac

2020-06-04 vs 2020-07-13 vs 2020-07-14 vs 2020-07-20

taxonomic databases 171 164 = lost 7 164 164 ok
living spp 1,829,672 1,831,489 1,821,132 1,821,020 = **lost 8,652**
extinct spp 38,145 38,121 38,121 38,120 = lost 25 
living infa 159,503 160,035 159,606 159,503 = lost 103
extinct infra 589 590 590 590
synonyms 1,710,621 1,691,500 = lost 19,121 1,689,866 = lost 20,755 1,689,833 = **lost 20,788**
common names 423,691 3,011 = lost 420,680 3,011 = lost 420,680 433,280 = **+9,589**

genera 175,459 184,526 175,285 175,284
families 10,758 12,267 10,874 10,873
orders 1,398 1,643 1,427 1,427
classes 322 387 328 328 
phyla 102 104 102 102 
kingdoms 7 7 7 7

**Comment by yroskov** [comment]
- [ ]  Unexpected disparities in GSD stats are identical to 2020-07-13, except Collembola.org (it's improved)

2020-06-04 Beta  vs  2020-07-20
3i Auchenorrhyncha: World Auchenorrhyncha Database 10451 vs 3210
Collembola.org 8555 = 8555
FLOW: Fulgoromorpha Lists On the WEB 13356 vs 13339  
Species Fungorum Plus 147898 vs 147934
The White-Files: Taxonomic checklist of the world’s whiteflies (Insecta: Hemiptera: Aleyrodidae) 1562 vs 1565
WoRMS Ostracoda: World Ostracoda Database 6307 vs 6308
WSC: World Spider Catalog 47391 vs 47714
WTaxa: Electronic Catalogue of Weevil names (Curculionoidea) 84512 vs 84476

**Comment by yroskov** [comment]
_**Probably, latest copies of GSDs have been illegally replaced with incorrect versions** (as Collembola in previous conversion)_

- [ ] **3i Auchenorrhyncha** (untouched): missing spp -7241; missing synonyms -10142

- [ ] **WSC** (untouched): increased number of spp +323; all synonyms missing -7110 

- [ ] **WoRMS Mollusca** (untouched): increased common names +7535 

- [ ] **World Ferns** (updated, but unlikely results): missing spp -1011; missing synonyms -5817

**Comment by yroskov** [comment]
**CoL 2020-07-22** at http://dev4.species.id:2204/col_plus/info/ac

2020-06-04 vs 2020-07-13 vs 2020-07-14 vs 2020-07-20 vs 2020-07-22

taxonomic databases 171 164 = lost 7 164 164 ok
living spp 1,829,672 1,831,489 1,821,132 1,821,020 = lost 8,652 1,820,906 = lost 8,766
extinct spp 38,145 38,121 38,121 38,120 38,120 = lost 25
living infa 159,503 160,035 159,606 159,503 159,603 = lost 100
extinct infra 589 590 590 590 590
synonyms 1,710,621 1,691,500 = lost 19,121 1,689,866 = lost 20,755 1,689,833 = lost 20,788 1,689,824 = lost 20,797
common names 423,691 3,011 = lost 420,680 3,011 = lost 420,680 433,280 = +9,589 433,280 = +9,589

genera 175,459 184,526 175,285 175,284
families 10,758 12,267 10,874 10,873
orders 1,398 1,643 1,427 1,427
classes 322 387 328 328
phyla 102 104 102 102
kingdoms 7 7 7 7

**Comment by gdower** [comment]
In World Ferns, it is family Thelypteridaceae that is missing approximately 1000 species:

![image](https://user-images.githubusercontent.com/8573609/88719682-4c0b7c80-d0e9-11ea-9817-a420f8ca208e.png)

The missing names are present in the dataset, so I thought it might be a sync issue:

https://data.catalogue.life/dataset/1140/classification?taxonKey=ee1a9e33-1cd2-5399-b11e-44e6ee1ad76e

The tribes have unusual names ([Amauropeltoid clade, Cyclosoroid clade](https://data.catalogue.life/dataset/1140/classification?taxonKey=ee1a9e33-1cd2-5399-b11e-44e6ee1ad76e)) but aren't flagged with an issue and sync completes successfully with no errors. These names have blocking decisions:

![image](https://user-images.githubusercontent.com/8573609/88720590-b8d34680-d0ea-11ea-8df4-437d7ca58222.png)

Since the [children are blocked, too](https://github.com/CatalogueOfLife/data/issues/156#issuecomment-656254722), that probably explains the missing ~1000 species. @yroskov, I unblocked them and applied a complex decision to rename them to exclude the word clade. Alternatively, we could take CoL ranks only.

![image](https://user-images.githubusercontent.com/8573609/88728242-e6be8800-d0f6-11ea-9fe4-40eae1663aab.png)

**Comment by yroskov** [comment]
WE cannot rename clades to tribes. Clade has no conventional rank. We need to delete those clade names.

**Comment by mdoering** [comment]
Clade names should not go into the CoL. To do so they should be flagged as name.type=INFORMAL. 
THis should be done either manually as the last resort in an update decision, or teaching the [name parser to recognize them automatically](https://github.com/gbif/name-parser/issues/68). Please do not add a block decision as this blocks the whole subtree!

**Comment by yroskov** [comment]
**CoL 2020-07-30** at http://dev4.species.id:9191/col_plus/

2020-06-04 Beta vs 2020-07-13 vs 2020-07-14 vs 2020-07-20 vs 2020-07-22 vs 2020-07-30

taxonomic databases 171 164 = lost 7 164 164 164 ok
living spp 1,829,672 1,831,489 1,821,132 1,821,020 = lost 8,652 1,820,906 = lost 8,766 1,829,341 = lost 331
extinct spp 38,145 38,121 38,121 38,120 38,120 38,145 = lost 0
living infa 159,503 160,035 159,606 159,503 159,603 159,729 = lost 100
extinct infra 589 590 590 590 590
synonyms 1,710,621 1,691,500 = lost 19,121 1,689,866 = lost 20,755 1,689,833 = lost 20,788 1,689,824 = lost 20,797
common names 423,691 3,011 = lost 420,680 3,011 = lost 420,680 433,280 = +9,589 433,280 = +9,589

genera 175,459 184,526 175,285 175,284 175,284
families 10,758 12,267 10,874 10,873 10,873
orders 1,398 1,643 1,427 1,427 1,427
classes 322 387 328 328 328
phyla 102 104 102 102 102
kingdoms 7 7 7 7 7

**Comment by yroskov** [comment]
Updated taxa/GSDs, 2020-06-04 Beta vs 2020-07-30:

ITIS Bacteria 9,980
ITIS Archaea 377

HymIS Pompilidae 4,632

**Comment by yroskov** [comment]
**HymIS Pompilidae needs flag Updated! (2)**

**Comment by gdower** [comment]
> **HymIS Pompilidae needs flag Updated! (2)**

FIXED

**Comment by yroskov** [comment]
**YR should invite **ITIS**, **WFerns**, HymIS & Scarabs before August release to check their metadata.** 

What's New is updated in GitHub.

**Comment by yroskov** [comment]
**CoL 2020-08-01** at http://dev4.species.id:9191/col_plus/

2020-06-04 Beta vs 2020-07-13 vs 2020-07-14 vs 2020-07-20 vs 2020-07-22 vs 2020-07-30 vs 2020-08-01

taxonomic databases 171 164 = lost 7 164 164 164 164 ok
living spp 1,829,672 1,831,489 1,821,132 1,821,020 = lost 8,652 1,820,906 = lost 8,766 1,829,341 = lost 331 1,829,341 = lost 331
extinct spp 38,145 38,121 38,121 38,120 38,120 38,145 38,145 = lost 0
living infa 159,503 160,035 159,606 159,503 159,603 159,729 159,729 = +226
extinct infra 589 590 590 590 590 590 590
synonyms 1,710,621 1,691,500 = lost 19,121 1,689,866 = lost 20,755 1,689,833 = lost 20,788 1,689,824 = lost 20,797 1,713,399 = +2,778 (?) 1,713,399 = +2,778
common names 423,691 3,011 = lost 420,680 3,011 = lost 420,680 433,280 = +9,589 433,280 = +9,589 433,354 =+9,663 (?) 433,354 =+9,663

genera 175,459 184,526 175,285 175,284 175,284 176,121 176,121 =+662 (?)
families 10,758 12,267 10,874 10,873 10,873 10,873 10,873
orders 1,398 1,643 1,427 1,427 1,427 1,427 1,427
classes 322 387 328 328 328 328 328
phyla 102 104 102 102 102 102 102
kingdoms 7 7 7 7 7 7 7

**No changes since 2020-07-30**

**Comment by yroskov** [comment]
Final polish:

- [ ] Update date-title in WhatsNew: 
AC_Interface/application/views/scripts/info/special.phtml  

- [ ] Update Info file  AC_Interface/application/views/scripts/info/ac.phtml 

- [x] Bibl. Citation for HymIS Pompilidae 
A.S. Kroupa; C. Schmid-Egger (2020). HymIS Pompilidae
Nomen.eumycetozoa.com
Lado; C. & Hernández-Crespo; J.C. (2020). Nomen.eumycetozoa.com
**Initials order corrected in metadata**

**Comment by yroskov** [comment]
CoL 2020-08-01 can be accepted as August release.
